wagtail/admin/wagtail_hooks.py

```diff
@@ -459,6 +459,36 @@ def register_core_features(features):
             'block_map': {'ordered-list-item': {'element': 'li', 'wrapper': 'ol'}}
         }
     })
+    features.register_editor_plugin(
+        'draftail', 'code-block', draftail_features.BlockFeature({
+            'type': 'code-block',
+            'icon': 'code',
+            'description': ugettext('Code Block'),
+        })
+    )
+    features.register_converter_rule('contentstate', 'code-block', {
+        'from_database_format': {
+            'code': BlockElementHandler('code-block'),
+        },
+        'to_database_format': {
+            'block_map': {'code-block': 'code'}
+        }
+    })
```

Comment:
We might need to have a look at whether `code-block` can be included or not. I wouldn't be surprised if there things that made it incompatible with Wagtail's rich text processing.

Reply:
Dropping `code-block` for now. We can look into this later.